test/services/test_post_service.py

```
@@ -187,4 +188,179 @@ async def test_view_post_success(db_session: Session):
 @pytest.mark.asyncio
 async def test_view_post_empty(db_session: Session):
     result = await view_post(db=db_session)
-    assert result == []
\ No newline at end of file
+    assert result == []
+
+def create_dummy_file(filename="test.jpg"):
```

Ruff (0.14.5)

Redefinition of unused `create_dummy_file` from line 18

(F811)